CruCentralCoast/Events/EventDetailsTransition.swift

```diff
@@ -0,0 +1,66 @@
+//
+//  EventDetailsTransition.swift
+//  CruCentralCoast
+//
+//  Created by Cam Stocker on 5/23/18.
+//  Copyright © 2018 Landon Gerrits. All rights reserved.
+//
+
+import Foundation
+import UIKit
+
+class EventDetailsTransition : NSObject, UIViewControllerAnimatedTransitioning {
+    private let duration = 0.5
+    var presenting = true
+    var originFrame = CGRect.zero
+    
```

Comment:
Reduce multiple empty lines to a single empty line

Reply:
fixed!